Makefile

```diff
@@ -6,7 +6,7 @@ ifeq ($(PROFILES),)
 PROFILES := 'pmm'
 endif
 
-env-up: 							## Start devcontainer.
+env-up: 							## Start devcontainer
```

Comment:
We have both variants in our Makefiles. I checked output of the `docker`, they don't use dots in help output